cmake/projects/tommath/hunter.cmake

```diff
@@ -19,7 +19,16 @@ hunter_add_version(
     SHA1
     733e4644a0b01260830a402b457ca82064fe30cd
 )
-
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0.1"
+    URL
+    "https://github.com/t0p4/libtommath/archive/v1.0.1-p1.tar.gz"
```

Comment:
Please use https://github.com/hunter-packages/libtommath/releases/tag/v1.0.1-p0 here.

Reply:
have done it